src/specter/network/SpecterInterface.php

```diff
@@ -116,17 +121,17 @@ public function putPacket(Player $player, DataPacket $packet, $needACK = false,
                     }
                     break;
                 case BatchPacket::class:
+                	/** @var BatchPacket $packet */
                     $packet->offset = 1;
                     $packet->decode();
 
                     $str = zlib_decode($packet->payload, 1024 * 1024 * 64); //Max 64MB
```

Comment:
I tested the plugin on my server and when I tried to make a specter, it kept giving an error message